python/cudf_polars/cudf_polars/experimental/base.py

```diff
@@ -42,8 +39,3 @@ def keys(self, node: Node) -> Iterator[tuple[str, int]]:
 def get_key_name(node: Node) -> str:
     """Generate the key name for a Node."""
     return f"{type(node).__name__.lower()}-{hash(node)}"
-
-
-def _concat(*dfs: DataFrame) -> DataFrame:
-    # Concatenate a sequence of DataFrames vertically
-    return Union.do_evaluate(None, *dfs)
```

Comment:
We don't need to move `_concat` to `utils.py` - It just "felt" like it belonged there.